.github/workflows/testAndPublish.yml

```diff
@@ -0,0 +1,279 @@
+    - name: Prepare for tests
+      run: ci/scripts/tests/beforeTests.ps1
+    - name: Cache scons build
+      uses: actions/cache/save@v4
+      with:
+        path: ${{ github.workspace }}
+        key: ${{ github.ref }}-${{ github.run_id }}
+
+  checkPot:
+    name: Check translator comments
+    runs-on: windows-latest
+    needs: buildNVDA
+    steps:
+    - name: Checkout cached build
+      uses: actions/cache/restore@v4
+      with:
+        path: ${{ github.workspace }}
+        key: ${{ github.ref }}-${{ github.run_id }}
+        fail-on-cache-miss: true
+    - name: Check comments for translators
+      run: ci/scripts/tests/translationCheck.ps1
+    - name: Upload artifact
+      if: ${{ success() || failure() }}
+      uses: actions/upload-artifact@v4
+      with:
+        name: makePot results
+        path: |
+          output/nvda.pot
+          output/potSourceFileList.txt
+        overwrite: true
+
+  licenseCheck:
+    name: Check license compatibility of dependencies
+    runs-on: windows-latest
+    needs: buildNVDA
+    steps:
+    - name: Checkout cached build
+      uses: actions/cache/restore@v4
+      with:
+        path: ${{ github.workspace }}
+        key: ${{ github.ref }}-${{ github.run_id }}
+        fail-on-cache-miss: true
+    - name: License check
+      run: ci/scripts/tests/licenseCheck.ps1
+
+  unitTests:
+    name: Run unit tests
+    runs-on: windows-latest
+    needs: buildNVDA
+    steps:
+    - name: Checkout cached build
+      uses: actions/cache/restore@v4
+      with:
+        path: ${{ github.workspace }}
+        key: ${{ github.ref }}-${{ github.run_id }}
+        fail-on-cache-miss: true
+    - name: Run unit tests
+      run: ci/scripts/tests/unitTests.ps1
+    - name: Replace relative paths in unit test results
+      if: ${{ success() || failure() }}
+      # Junit reports fail on these
+      shell: bash
+      run: sed -i 's|file="..\\|file="|g' testOutput/unit/unitTests.xml
+    - name: Upload artifact
+      if: ${{ success() || failure() }}
+      uses: actions/upload-artifact@v4
+      with:
+        name: Unit tests results
+        path: testOutput/unit/unitTests.xml
+        overwrite: true
+    - name: Publish unit test report
+      uses: mikepenz/action-junit-report@v5
+      if: ${{ success() || failure() }}
+      with:
+        check_name: Unit tests
+        detailed_summary: true
+        annotate_only: true
+        report_paths: testOutput/unit/unitTests.xml
+
+  createLauncher:
+    name: Create launcher
+    runs-on: windows-latest
+    needs: buildNVDA
+    steps:
+    - name: Checkout cached build
+      uses: actions/cache/restore@v4
+      with:
+        path: ${{ github.workspace }}
+        key: ${{ github.ref }}-${{ github.run_id }}
+        fail-on-cache-miss: true
+    - name: Set version variables
+      run: ci/scripts/setBuildVersionVars.ps1
+    - name: Set scons args
+      run: ci/scripts/setSconsArgs.ps1
+    - name: Prepare source code for launcher
```

Comment:
Do we actually need this one if we explicitly run `scons launcher`
In other words, is there anything a bare run of scons builds that scons launcher doesn't?

Reply:
Generally I think no - but err on keeping it to keep consistency with the AppVeyor script.
For example, if `feature_buildAppx` is enable, we will build Windows Store version components that would otherwise not get included in the launcher